src/main/java/org/moltimate/moltimatebackend/dto/Alignment/AlignmentMotifResponse.java

```diff
@@ -0,0 +1,80 @@
+package org.moltimate.moltimatebackend.dto.Alignment;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.biojava.nbio.structure.Structure;
+import org.moltimate.moltimatebackend.model.Alignment;
+import org.moltimate.moltimatebackend.model.Motif;
+import org.moltimate.moltimatebackend.model.Residue;
+import org.moltimate.moltimatebackend.util.StructureUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A list of Alignments and useful data around them.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AlignmentMotifResponse {
+
+    private String motifPdbId;
+    private String motifEcNumber;
+    private List<Residue> activeSiteResidues;
+
+    List<SuccessfulAlignment> alignments; // Alignments that found a match
```

Comment:
@blackpan2  ++ might be something worth mentioning in docs/hand off that 'language' we use to avoid problems like understanding a bad result vs it didnt work.